ui/lib/core/addon/components/form-field.hbs

```diff
@@ -133,12 +133,15 @@
       @value={{get @model this.valuePath}}
       @onChange={{this.setAndBroadcast}}
       @label={{this.labelString}}
-      @labelClass="title {{if (eq @mode 'create') 'is-5' 'is-4'}}"
+      @labelClass={{unless @attr.options.shouldOnlyShowSingleRow (if (eq @mode "create") "title is-5" "title is-4")}}
       @helpText={{@attr.options.helpText}}
       @subText={{@attr.options.subText}}
       @onKeyUp={{this.handleKeyUp}}
+      @keyPlaceholder={{@attr.options.keyPlaceholder}}
+      @valuePlaceholder={{@attr.options.valuePlaceholder}}
       @validationError={{this.validationError}}
-      class="form-section"
+      @shouldOnlyShowSingleRow={{@attr.options.shouldOnlyShowSingleRow}}
+      class={{unless @attr.options.shouldOnlyShowSingleRow "form-section"}}
```

Comment:
this was also changed in my PR to the following (`isSectionHeader` is already an attr option we use for search select styling)
```js
      class={{if @attr.options.isSectionHeader "form-section"}}